test-requirements.txt

```diff
@@ -1,7 +1,9 @@
 -r jenkins/requirements.txt
 -r aws/requirements.txt
+-r release/test/requirements.txt
```

Comment:
I think this needs to be `release/requirements.txt`. Travis is failing on this step at the moment.